spaceship.zsh

```diff
@@ -309,6 +314,23 @@ spaceship_swift_version() {
   fi
 }
 
+# GOLANG
+# Show current version of golang.
+spaceship_golang_status() {
```

Comment:
Take a look how it's implemented in other themes. For example, [bullet-train-oh-my-zsh-theme](https://github.com/caiogondim/bullet-train-oh-my-zsh-theme/blob/master/bullet-train.zsh-theme#L527-L539). Also, that would be a cool feature if we will support [Go Version Manager](https://github.com/moovweb/gvm) or other tool for managing Go version. Will you consider this?